learn/get-started/create-account.md

```diff
@@ -0,0 +1,135 @@
+
+<example name="Creating a test account">
+```sh
+$ curl "https://horizon-testnet.stellar.org/friendbot?addr=GDGOKHIRX63EIAXKVI77BQV7LMDUH7DR4BMDDU77DJUXLPUU5HAXGN64"
+```
+
+```js
+// The SDK does not have tools for creating test accounts, so you'll have to
+// make your own HTTP request.
+var request = require('request');
+request.get({
+  url: 'https://horizon-testnet.stellar.org/friendbot',
+  qs: { addr: pair.accountId() },
+  json: true
+}, function(error, response, body) {
+  if (error || response.statusCode !== 200) {
+    console.error('ERROR!', error || body);
+  }
+  else {
+    console.log('SUCCESS! You have a new account :)\n', body);
+  }
+});
+```
+
+```java
+// The SDK does not have tools for creating test accounts, so you'll have to
+// make your own HTTP request.
+import java.net.*;
+import java.io.*;
+import java.util.*;
+
+String friendbotUrl = String.format(
+  "https://horizon-testnet.stellar.org/friendbot?addr=%s",
+  pair.getAccountId());
+InputStream response = new URL(friendbotUrl).openStream();
+String body = new Scanner(response, "UTF-8").useDelimiter("\\A").next();
+System.out.println("SUCCESS! You have a new account :)\n" + body);
+```
+</example>
+
+Now for last step: getting the account’s details and checking its balance! Accounts can carry multiple balances—one for each type of currency they hold.
+
+<example name="Getting account details">
+```sh
+curl "https://horizon-testnet.stellar.org/accounts/GCRDH24DCTMKRL3SESRQ4QRKHJ56XGAJBQHHXXED3RTBQTBC36RCX4JI"
+```
+
+```js
+var server = new StellarSdk.Server('https://horizon-testnet.stellar.org');
+
+// the JS SDK uses promises for most actions, such as retrieving an account
+server.accounts().accountId(pair.accountId()).call().then(function(account) {
+  console.log('Balaces for account: ' + pair.accountId());
+  account.balances.forEach(function(balance) {
+    console.log('Type:', balance.asset_type, ', Balance:', balance.balance);
+  });
+});
+```
+
+```java
+Server server = new Server("https://horizon-testnet.stellar.org");
+AccountResponse account = server.accounts().account(pair.getAccountId());
+System.out.println("Balances for account " + pair.getAccountId());
+for (AccountResponse.Balance balance : account.getBalances()) {
+  System.out.println(String.format(
+    "Type: %s, Code: %s, Balance: %s",
+    balance.getAssetType(),
+    balance.getAssetCode(),
+    balance.getBalance()));
+}
+```
+</example>
+
+Now that you’ve got an account you can [start making and receiving payments](transactions.html).
+
+<a class="button button--previous" href="index.html">Back</a>
+<a class="button button--next" href="transactions.html">Next</a>
+
+
+
+[1]: You can change an account’s public key later, but the account ID will always stay the same. When you retrieve an account’s data, you’ll be able to get both its ID and its public key.
```

Comment:
That is a horizon thing. He gives each object a unique "id". For accounts it happens to be the account_id. For txs I think it is the hash of the tx etc.